src/commands/registerCommands.ts

```diff
@@ -41,6 +42,7 @@ export function registerCommands(): void {
 
     // github
     registerCommandWithTreeNodeUnwrapping('containerApps.connectToGitHub', connectToGitHub);
+    registerCommandWithTreeNodeUnwrapping('containerApps.disconnectRepo', disconnectRepo);
```

Comment:
`containerApps.disconnectGitHub`? Again, kind of weird to me to not use `connectToRepo` but use `disconnectRepo`.